class-wc-settings-page-komoju.php

```diff
@@ -40,21 +45,28 @@ public function __construct()
     public function get_sections()
     {
         $sections = [
-            '' => __('KOMOJU account settings', 'komoju-woocommerce'),
+            ''             => __('Payment methods', 'komoju-woocommerce'),
+            'api_settings' => __('API settings', 'komoju-woocommerce'),
```

Comment:
This splits our settings page into 2 sub-pages. The first one is just a sign-into-komoju button and payment methods. The second contains detailed fields that 90% of merchants do not need to touch, like API key, webhook secret, komoju endpoint, etc.